src/Persistent/Api.hs

```diff
@@ -126,12 +128,18 @@ emptyAulaData = AulaData nil nil nil nil nil 21 21 30 3 0
 -- FIXME move enough specialized calls to IO in PersistM to remove MonadIO
 class MonadIO m => PersistM m where
     getDb :: AulaGetter a -> m a
+
+class PersistM m => UpdatePersistM m where
     modifyDb :: AulaSetter a -> (a -> a) -> m ()
```

Comment:
In my opinion this could be `modifyDb :: User -> AulaSetter a -> (a -> a) -> m ()`